examples/tutorials/advance_model_config/03_train_steps.ipynb

```diff
@@ -0,0 +1,443 @@
+{
```

Comment:
<blockquote><code>train_steps</code> is a mapping from names (more on that later) to parameters of that training procedure. ...</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>